option/internaloption/internaloption.go

```diff
@@ -209,6 +209,27 @@ func (w enableNewAuthLibrary) Apply(o *internal.DialSettings) {
 	o.EnableNewAuthLibrary = bool(w)
 }
 
+// EnableAsyncRefreshDryRun  returns a ClientOption that specifies if libraries in this
+// module should asynchronously refresh auth token in parallel to sync refresh
+//
+// errHandler function will be called when there is an error while refreshing
+// the token asynchronously
+//
+// This is an experimental option and will be removed in the future
```

Comment:
The three sentences above should all end with a period (`.`).

Reply:
I think "experimental" should be in all caps: "EXPERIMENTAL".

Reply:
Done